ui/src/views/Tasks/ViewTask/index.jsx

```diff
@@ -949,12 +950,16 @@ export default class ViewTask extends Component {
             <br />
             <Grid container spacing={3}>
               <Grid item xs={12} md={6}>
-                <TaskDetailsCard
-                  task={task}
-                  dependentTasks={dependentTasks}
-                  dependents={dependents}
-                  onDependentsPageChange={this.handleDependentsPageChange}
-                />
+                <AuthContext.Consumer>
+                  {auth => (
```

Comment:
injecting Auth here because task dependencies would be fetched from the queue API directly which might require credentials on some deployments (non-public)